smoke_tests/tests/smoke_test_case.py

```diff
@@ -0,0 +1,27 @@
+import re
+from functools import cache
+from unittest import TestCase
+from urllib.parse import urljoin
+
+import requests
+from requests import Response
+
+
+class SmokeTestCase(TestCase):
```

Comment:
Not a huge deal but I generally prefer pytest's decorator-based approach to unittesting vs. the class-based approach w/unittest. Something for tech talk another time.